media-art-background.js

```diff
@@ -78,17 +78,17 @@ function setBackground(root, appLayout, lovelace, bgroundElem) {
 
 // get HA root element
 let root = document.querySelector("home-assistant");
-root = root.shadowRoot.querySelector("home-assistant-main").shadowRoot.querySelector("app-drawer-layout partial-panel-resolver ha-panel-lovelace").shadowRoot.querySelector("hui-root");
+root = root.shadowRoot.querySelector("home-assistant-main").shadowRoot.querySelector("ha-drawer partial-panel-resolver ha-panel-lovelace").shadowRoot.querySelector("hui-root");
 
 // get constant elements from HA root element
-const appLayout = root.shadowRoot.querySelector("ha-app-layout");
+const appLayout = root.shadowRoot;
 const lovelace = root.lovelace;
 
 // create container element, set style and append to container
 const bgroundElem = document.createElement("div"); // create empty container for background 
 setupStyle(lovelace, bgroundElem);
 appLayout.appendChild(bgroundElem);
-appLayout.shadowRoot.querySelector("#contentContainer").style.transform = "none";
+//appLayout.shadowRoot.querySelector("#contentContainer").style.transform = "none";
```

Comment:
Does this break something that you removed it? If so, let's just delete the line completely instead of comment it out.